src/components/PDF/Applications.tsx

```diff
@@ -139,43 +146,41 @@ class Applications extends Component<Props, State, {}> {
     );
   }
 
-  changeCurrentPage = (newCurrentPage: number) => {
-    this.setState({ currentPage: newCurrentPage }, this.getDocuments);
-  }
-
-  onChangeViewPermission = (event: any) => {
-    const {
-      currentUser,
-    } = this.state;
-    currentUser.viewPermission = event.target.ischecked;
-    this.setState({ currentUser }, this.getDocuments);
-  }
-
-  getDocuments = () => {
-    const {
-      itemsPerPageSelected,
-    } = this.state;
-    const itemsPerPage = Number(itemsPerPageSelected.value);
-    // fetch call here to get all the current Documents to fill
-  }
+  // TODO: Pagination
+  // changeCurrentPage = (newCurrentPage: number) => {
+  // this.setState({ currentPage: newCurrentPage }, this.getDocuments);
+  // }
+  // }
+  //
+  // handleChangeItemsPerPage = (itemsPerPageSelected: any) => {
+  //  this.setState({
+  //    currentPage: 0,
+  //  });
+  // }
+  // getDocuments = () => {
+  //   const {
+  //     itemsPerPageSelected,
+  //   } = this.state;
+  //   const itemsPerPage = Number(itemsPerPageSelected.value);
+  //   // fetch call here to get all the current Documents to fill
+  // }
```

Comment:
do we need to keep this? or can we just delete

Reply:
It might be useful for the future